notebooks/instructlab-knowledge/utils/create_seed_dataset.py

```diff
@@ -19,12 +19,27 @@ def get_seed_dataset(path: str) -> Dataset:
                       of seed data for the knowledge generation pipeline in
                       SDG.
     """
-    valid_path = is_dir_valid(path)
-    ds = create_dataset_from_dir(valid_path)
+    if not chunks_path.is_dir():
+        raise ValueError(f"Path to chunks {chunks_path} must be a directory")
+    if not seed_examples_path.is_dir():
+        raise ValueError(f"Path to seed examples {seed_examples_path} must be a directory")
+
+    files = list(seed_examples_path.iterdir())
+    has_qna = any(f.name == 'qna.yaml' for f in files)
+    files = list(chunks_path.iterdir())
+    has_chunks_jsonl = any(f.name == 'chunks.jsonl' for f in files)
+
+    if not has_qna:
+        raise ValueError(f"Seed examples dir {seed_examples_path} does not contain a qna.yaml")
+
+    if not has_chunks_jsonl:
+        raise ValueError(f"Chunks dir {chunks_path} does not contain a chunks.jsonl")
+
+    ds = create_dataset_from_dir(chunks_path, seed_examples_path)
 
     return ds
 
-def is_dir_valid(path: str) -> Path:
+def is_dir_valid(path: str) -> None:
```

Comment:
nit:
```suggestion
def validate_dir(path: str) -> Path:
```
not sure why the return signature was changed from `Path` to `None` here, but probably could use some renaming